src/games/Lexible/models/PresenterModel.ts

```diff
@@ -624,39 +592,80 @@ export class LexiblePresenterModel extends ClusterfunPresenterModel<LexiblePlaye
                 block.setScore( Math.max(word.length, block.score), player.teamName);  
                 placedLetters.push(l);                
             }
-
+            // however, do mark redundant word submissions
+            if (word.length === block.score && block.team === player.teamName) {
+                placedLetters.push(l);
+            }
         })
 
         if(word.length > player.longestWord.length) player.longestWord = word;
 
-        this.sendToEveryone((p, isExited) => {
-            return new LexibleScoredWordMessage({
-                sender: this.session.personalId,
-                scoringPlayerId: player.playerId,
-                team: player.teamName,
+        this.requestEveryone(LexibleBoardUpdateEndpoint, (p, isExited) => {
+            return {
                 letters: placedLetters,
-                score: word.length
-            }) 
-        })
+                score: word.length,
+                scoringPlayerId: player.playerId,
+                scoringTeam: player.teamName
+            }
+        }, true);
+
         this.invokeEvent(LexibleGameEvent.WordAccepted, word.toLowerCase(), player)
         if (player.teamName === "A" || player.teamName === "B") {
             this.checkForWin();
         } else {
             Logger.warn("WEIRD: Player with unknown teamname")
         }
+        this.saveCheckpoint();
     }
 
-    // -------------------------------------------------------------------
-    //  handlePlayerWordSubmit 
-    // -------------------------------------------------------------------
-    handlePlayerWordSubmit = (playerId: string, data: WordSubmissionData) => {
-        if(!data) throw Error("handlePlayerWordSubmit: No data")
-        const player = this.players.find(p => p.playerId === playerId);
+    handleOnboardClient = (sender: string, message: unknown): LexibleOnboardClientMessage => {
```

Comment:
I like this communication style and how it enables functions like this that don't have to think about the external messaging system.  VERY NICE

Reply:
Thank you!